Exercise_D.js

```diff
@@ -14,7 +14,19 @@
 Use a combination of conditionals and loops to create your answer for this question. */
 
 function multiplyNumber(number) {
-  // CODE HERE
+
+  if (number <= 0 || number == 0 || typeof number != "number") {
+    return "ERROR";
+  }
+  else if (number >= 1000000) {
+    return number;
+  }
+  else {
+    while (number < 1000000) {
+      number = number * 10;
+    }
+    return number;
+  }
 }

```

Comment:
Great implementation here 
only 'improvement'
short form of line 26:
number *= 10